rules/rbe_repo.bzl

```diff
@@ -689,8 +697,8 @@ _rbe_autoconfig = repository_rule(
             doc = ("Optional. The location of java_home in the container. " +
                    "For example, '/usr/lib/jvm/java-8-openjdk-amd64'. If " +
                    "not set, the rule will attempt to read the JAVA_HOME env " +
-                   "var from the container. If that is not set the rule will " +
-                   "fail."),
+                   "var from the container. If set to a blank string, the " +
+                   "rule will skip java config generation."),
```

Comment:
I think we need to add a new boolean attr (as you also mentioned on chat). Particularly because this would not have any equivalent for the cc config. So yes, lets add java_config (and later on cc_config) as boolean attr (or some other name if you have a suggestion). Default value set to true. If set to false, and java_home is not None, error out.

Reply:
Done